api/awsauth/README.md

```diff
@@ -13,46 +13,28 @@ npm install -g serverless
 
 ## Deploy
 
-Currently we have a `dev` and `prod` stage.
+Currently we have a `dev` and `prod` stage deployed via Github Actions.
 
-Deploys are set up to look for environment variables in a `.env` file.
+A .env file is used to configure most of the environment variables for deploy.
```

Comment:
I don't quite understand how 'prod' and 'dev' end up doing different things.  Does the generated .env file have separate configurations for them?  What specifically is different to differentiate prod and env?  Mostly the CLOUDFRONT_DISTRIBUTION_ID?

If it's easy to add a bit more context about how the two environments work, that'd be nice, because it's a bit opaque just looking at the breadcrumbs I can see in this PR at least.